Unit Test "EmailTest.test_invalid_values_check" is failing

### Summary
The unit test case "EmailTest.test_invalid_values_check" fails because the `not-found .fr` was registered a few weeks ago and doesn't throw a `ValueError` exception anymore. This affects the CI process. ([Tests #2026](https://github.com/python-restx/flask-restx/actions/runs/8105255279))

For test cases, reserved domain names should be used instead, as described in [RFC 2606](https://datatracker.ietf.org/doc/html/rfc2606#section-2).

### Code
The `EmailTest.test_invalid_values_check` testcase expects all the domain names to be invalid although the `not-found .fr` one is actually valid. ([ICANN Lookup](https://lookup.icann.org/en/lookup))
```python
    @pytest.mark.parametrize(
        "value",
        [
            "coucou@not-found.fr",
            "me@localhost",
            "me@127.0.0.1",
            "me@127.1.2.3",
            "me@::1",
            "me@200.8.9.10",
            "me@2001:db8:85a3::8a2e:370:7334",
        ],
    )
    def test_invalid_values_check(self, value):
        email = inputs.email(check=True)
        self.assert_bad_email(email, value)
```

### Expected Behavior
All unit tests should be passing.

### Actual Behavior
The `EmailTest.test_invalid_values_check` testcase fails.

### Error Messages/Stack Trace
Here's a partial result from `tox` command.
```
======================================================= FAILURES =======================================================
_______________________________ EmailTest.test_invalid_values_check[coucou@not-found.fr] _______________________________

self = <tests.test_inputs.EmailTest object at 0x7f5816d65150>, value = 'coucou@not-found.fr'

    @pytest.mark.parametrize(
        "value",
        [
            "coucou@not-found.fr",
            "me@localhost",
            "me@127.0.0.1",
            "me@127.1.2.3",
            "me@::1",
            "me@200.8.9.10",
            "me@2001:db8:85a3::8a2e:370:7334",
        ],
    )
    def test_invalid_values_check(self, value):
        email = inputs.email(check=True)
>       self.assert_bad_email(email, value)

tests/test_inputs.py:667:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <tests.test_inputs.EmailTest object at 0x7f5816d65150>
validator = <flask_restx.inputs.email object at 0x7f5816d648b0>, value = 'coucou@not-found.fr'
msg = '{0} is not a valid email'

    def assert_bad_email(self, validator, value, msg=None):
        msg = msg or "{0} is not a valid email"
>       with pytest.raises(ValueError) as cm:
E       Failed: DID NOT RAISE <class 'ValueError'>

tests/test_inputs.py:605: Failed
```

### Environment
- Python version: py 3.8 - 3.12, pypy 3.8
- Flask version: 2, 3
- Flask-RESTX version: 1.3.0
- Other installed Flask extensions: none

### Additional Context
- This issue will be fixed by https://github.com/python-restx/flask-restx/pull/603

--- a/flask_restx/fields.py
+++ b/flask_restx/fields.py
@@ -333,9 +333,11 @@ class List(Raw):
         return [
             self.container.output(
                 idx,
-                val
-                if (isinstance(val, dict) or is_attr(val)) and not is_nested
-                else value,
+                (
+                    val
+                    if (isinstance(val, dict) or is_attr(val)) and not is_nested
+                    else value
+                ),
             )
             for idx, val in enumerate(value)
         ]

--- a/flask_restx/marshalling.py
+++ b/flask_restx/marshalling.py
@@ -171,9 +171,11 @@ def _marshal(data, fields, envelope=None, skip_none=False, mask=None, ordered=Fa
         return (key, value)
 
     items = (
-        (k, marshal(data, v, skip_none=skip_none, ordered=ordered))
-        if isinstance(v, dict)
-        else __format_field(k, v)
+        (
+            (k, marshal(data, v, skip_none=skip_none, ordered=ordered))
+            if isinstance(v, dict)
+            else __format_field(k, v)
+        )
         for k, v in fields.items()
     )
 

--- a/flask_restx/namespace.py
+++ b/flask_restx/namespace.py
@@ -253,9 +253,11 @@ class Namespace(object):
         def wrapper(func):
             doc = {
                 "responses": {
-                    str(code): (description, [fields], kwargs)
-                    if as_list
-                    else (description, fields, kwargs)
+                    str(code): (
+                        (description, [fields], kwargs)
+                        if as_list
+                        else (description, fields, kwargs)
+                    )
                 },
                 "__mask__": kwargs.get(
                     "mask", True

--- a/flask_restx/schemas/__init__.py
+++ b/flask_restx/schemas/__init__.py
@@ -4,6 +4,7 @@ and allows to validate specs against them.
 
 .. versionadded:: 0.12.1
 """
+
 import io
 import json
 

--- a/tasks.py
+++ b/tasks.py
@@ -125,9 +125,11 @@ def benchmark(
         "--benchmark-max-time={0}".format(max_time),
         "--benchmark-autosave" if save else None,
         "--benchmark-compare" if compare else None,
-        "--benchmark-histogram=histograms/{0:%Y%m%d-%H%M%S}".format(ts)
-        if histogram
-        else None,
+        (
+            "--benchmark-histogram=histograms/{0:%Y%m%d-%H%M%S}".format(ts)
+            if histogram
+            else None
+        ),
         "--benchmark-cprofile=tottime" if profile else None,
     )
     cmd = "pytest tests/benchmarks {0}".format(kwargs)

--- a/tests/test_inputs.py
+++ b/tests/test_inputs.py
@@ -653,7 +653,7 @@ class EmailTest(object):
     @pytest.mark.parametrize(
         "value",
         [
-            "coucou@not-found.fr",
+            "coucou@not-found.test",
             "me@localhost",
             "me@127.0.0.1",
             "me@127.1.2.3",

